testing/cli_test.go

```diff
@@ -97,7 +98,7 @@ func TestVestingAccounts(t *testing.T) {
 	assert.Equal(t, vest1Addr, gotAddr)
 	amounts := accounts[0].Get("base_vesting_account.original_vesting").Array()
 	require.Len(t, amounts, 1)
-	assert.Equal(t, "utgd", amounts[0].Get("denom").String())
+	assert.Equal(t, app.BaseCoinUnit, amounts[0].Get("denom").String())
```

Comment:
nit: I like test to be self containing if possible. `BaseCoinUnit` is an extra click compared to some minor duplication ;-)